docs/contribution_details/CONTRIBUTING_GUIDE.md

```diff
@@ -0,0 +1,28 @@
+# Contributing to LibreLingo
+
+### Contribute to course material
+If you are interested in contributing to course development, please fill the [this](https://danielkantor196881.typeform.com/to/V00Paz) form. The project is in a very early stage right now, so you might not be able to contribute right away. Your work will be released in this GitHub repository and you will show up as a contributor here.
+
+
+### Development and other forms of contribution
+
+Check the latest **Development documentation** on [Read the Docs](https://librelingo.readthedocs.io/en/latest/)
```

Comment:
This line should not be necessary here, because the person will already be on that page almost certainly

Reply:
> This line should not be necessary here, because the person will already be on that page almost certainly

I was thinking if a person could find the link for the development docs in the same file it might be easier for him to read the other contents inside the same file. Like over here if the person is under `CONTRIBUTING GUIDE` he can find all the necessary material under one file. Thoughts on this @kantord ?

Reply:
I think you should just be able to link to that markdown file. mkdocs should take care of the rest.

Reply:
mkdocs converts these .md files into a static webpage. that webpage is hosted at https://librelingo.readthedocs.io/en/latest, but since the person reading it is already on that page, we don't need to tell them.